extensions.json

```diff
@@ -12,5 +12,19 @@
     "sha256sum": "6fb9647d5e202f2e030cd16a36706a43481d5065a54669967db68690e2199338",
     "foxe": "https://github.com/foxglove/studio-extension-turtlesim/releases/download/v0.0.1/foxglove.studio-extension-turtlesim-0.0.1.foxe",
     "keywords": ["ros","turtlesim","turtle","simulator","tutorial"]
+  },
+  {
+    "id": "uwprl.studio-extension-mushr",
+    "name": "mushr",
+    "description": "Custom buttons for differentiating publish types for the MuSHR base package",
+    "publisher": "uwprl",
+    "homepage": "https://github.com/mcdoerr/foxglove-mushr-extension",
+    "readme": "https://github.com/mcdoerr/foxglove-mushr-extension/blob/main/README.md",
+    "changelog": "https://github.com/mcdoerr/foxglove-mushr-extension/blob/main/CHANGELOG.md",
+    "license": "UNLICENSED",
+    "version": "0.0.0",
+    "sha256sum": "",
```

Comment:
You can get this value via `shasum -a 256 uwprl.MushrTeleopButtons-0.0.0.foxe`

Reply:
Thanks for the info! Should be updated now!